lib/backend/k8s/conversion/conversion.go

```diff
@@ -400,6 +400,9 @@ func (c converter) k8sRuleToCalico(rPeers []networkingv1.NetworkPolicyPeer, rPor
 		if p.Protocol != nil {
 			protval := kapiv1.Protocol(fmt.Sprintf("%s", *p.Protocol))
 			port.Protocol = &protval
+		} else {
```

Comment:
I think this makes line 398 redundant so you can remove that line and its comment

Reply:
👍